app/views/static_pages/index.html.erb

```diff
@@ -1,3 +1,13 @@
-<h1 class="col-10 offset-1">
-	Welcome
-</h1>
+
+
+<div class="booyah-box col-4">
+<h1>Join a Game</h1>
+
+<% @games.available.each do |game| %>
+    <% if game.white_player %>
+      <%= game.name + "'s game" %>
```

Comment:
Is this supposed to be `<%= game.white_player.name + "'s game" %>`?

Reply:
No game .name  works and is what i wanted